go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/ceph/ceph-csi
 
-go 1.22
+go 1.22.0
```

Comment:
if we are sticking to x.y.z version here we can remove the toolchain below and keep only one version of golang not two different version, we got into a build problem in downstream for using x.y.x, please check that one (might need some other downstream specific patch)

Reply:
ok, this is a try with `1.22` without minor release included. Would that be fine?

Reply:
yes, we will be good but i dont think it works.  we can check the downstream issue later as this is changed with go 1.22

Reply:
@nixpanic let's remove the toolchain and set go : 1.22.0?

Reply:
yes the above is also fine, if that works :)

Reply:
It seems to work when commit aa8be9b5f8424b68136cf813c7c7a22c14b8d230 is included.